src/spikeinterface/curation/tests/common.py

```diff
@@ -82,6 +84,31 @@ def sorting_analyzer_with_splits():
     return make_sorting_analyzer_with_splits(sorting_analyzer)
 
 
+def make_trained_pipeline():
```

Comment:
Make it a fixture? So we don't need the if trained folder and it always returns the model folder